zebra-state/src/in_memory.rs

```diff
@@ -105,6 +105,7 @@ impl Service<Request> for InMemoryState {
 
                 async move { Ok(Response::BlockLocator { block_locator }) }.boxed()
             }
+            Request::AwaitUtxo(_) => unimplemented!("this Service is being removed"),
```

Comment:
Now that #896 is resolved we're clear to remove this, right?

Reply:
noted that #864 is unblocked now: https://github.com/ZcashFoundation/zebra/issues/864#issuecomment-683959268

Reply:
I'll go ahead and remove the in memory service now and then rebase this change onto that PR.